pkg/packagekit/updates.less

```diff
@@ -1,21 +1,27 @@
-@import "/page.css";
+@import (less) "../lib/page.css";
```

Comment:
I now think this should be removed completely.  Sorry for the wrong advice earlier.

Nothing in updates.less needs anything from page.css, and by having it here causes page.css to be included twice.